### PR TITLE
Update API compat version number to 1.1.0 for stable release packages

### PR DESCRIPTION
**Pull request by codemzs**

### CODE REVIEW

File: tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj [hunk 1]
```diff
@@ -7,13 +7,13 @@
...
-    <PackageReference Include="Microsoft.ML.LightGbm" Version="1.0.0" />
-    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="1.0.0" />
-    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="1.0.0" />
+    <PackageReference Include="Microsoft.ML" Version="1.1.0" />
+    <PackageReference Include="Microsoft.ML.DataView" Version="1.1.0" />
+    <PackageReference Include="Microsoft.ML.CpuMath" Version="1.1.0" />
+    <PackageReference Include="Microsoft.ML.FastTree" Version="1.1.0" />
+    <PackageReference Include="Microsoft.ML.LightGbm" Version="1.1.0" />
+    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="1.1.0" />
+    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="1.1.0" />
```
Comment (artidoro) [inline tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj]: Are you adding TensorFlow as well?

File: tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj [hunk 2]
```diff
@@ -7,13 +7,13 @@
...
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ML" Version="1.0.0" />
-    <PackageReference Include="Microsoft.ML.DataView" Version="1.0.0" />
-    <PackageReference Include="Microsoft.ML.CpuMath" Version="1.0.0" />
-    <PackageReference Include="Microsoft.ML.FastTree" Version="1.0.0" />
-    <PackageReference Include="Microsoft.ML.LightGbm" Version="1.0.0" />
-    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="1.0.0" />
-    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="1.0.0" />
+    <PackageReference Include="Microsoft.ML" Version="1.1.0" />
```
Comment (artidoro) [inline tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj]: I am curious to know how other teams do it, but are we going to keep increasing the reference packages to the latest GA?

File: tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj (hunk 1 above)
Comment (artidoro) [reply tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj]: Not in 1.1 GA.

File: tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj (hunk 2 above)
Comment (codemzs) [reply tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj]: I would imagine so. @eerhardt thoughts?

Comment (eerhardt) [reply tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj]: That would be my plan. Once you ship `1.0`, you don't want to break any of those APIs. Then you ship new APIs in `1.1`. Now as you are building `1.2` you don't want to break those new APIs you just added in `1.1`.

Comment (artidoro) [reply tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj]: Yes that's what I thought! It makes total sense. Thanks @eerhardt

Comment (artidoro) [reply tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj]: @codemzs If possible we should add this to the notes on how to release a new version.

**Pull request merged**